core/src/driver/scheduler.rs

```diff
@@ -36,13 +40,18 @@ impl AuctionTimingConfiguration {
     /// invariants must hold:
     /// - `target_start_solve_time < solver_time_limit`
     /// - `solver_time_limit < SOLVING_WINDOW`
+    /// - `min_solution_submit_time < SOLVING_WINDOW`
     ///
     /// Where `SOLVING_WINDOW` represents the amount of time within a batch in
     /// which a solution is accepted. There is an amount of time at the end of a
     /// batch where solutions are no longer accepted, this is done to allow
     /// traders time to make decisions after the previous batch has already
     /// finalized.
-    pub fn new(target_start_solve_time: Duration, solver_time_limit: Duration) -> Self {
+    pub fn new(
+        target_start_solve_time: Duration,
+        solver_time_limit: Duration,
+        min_solution_submit_time: Duration,
```

Comment:
Conceptually, I'd find it easier to understand if we had an `earliest_solution_submission_time` and a `latest_solution_submission_time`. The `solver_time_limit` then follows from `latest_solution_submission_time`.

Reply:
I agree with this. Renamed it in AuctionTimingConfiguration and in the Options struct. I did leave the env variable as SOLVER_TIME_LIMIT to not cause issues with our configuration. Let me know if you're ok with these changes @fleupold .

Reply:
Yes changes look good. I would however prefer if the env variable could also be renamed (as this is what consumers of our binary need to understand when running it). As for configuration changes, we can create a staging PR that has both values (old and new) set to the same value and then remove the old one once the PR is merged.

For prod, we usually make sure when we deploy that we adjust the configs to have only the "expected" diff to staging (e.g. gurobi for standard solver).

Maybe we can do this in a separate PR and tag it with [ConfigChange] so that whoever does the next release is reminded about this?

Reply:
Created a [`config change` label](https://github.com/gnosis/dex-services/labels/config%20change). I would also prefer renaming the environment variable in this case.

Reply:
The issue with github labels is that they don't show up when compiling the changelog for a release (which is the point at which we should remember to update the prod env var as well). That's why I tend to prefer good old ASCI title prefixes as  labels.

Reply:
Oh shoot good point! I like the GitHub labels cause they are so colourful (here the dark red really catches the attention).